js/webviews.js

```diff
@@ -23,6 +23,23 @@ function onPageLoad (e) {
     var tab = _this.getAttribute('data-tab')
     var url = _this.getAttribute('src') // src attribute changes whenever a page is loaded
 
+    if (webviews.get(tabs.getSelected()).canGoBack()) {
+      goBackButton.classList.remove('fa-angle-right')
+      goBackButton.classList.add('fa-angle-left')
+      goBackButton.style.opacity = '1'
+      goBackButton.title = l('goBack')
+    } else if (webviews.get(tabs.getSelected()).canGoForward()) {
+      goBackButton.classList.remove('fa-angle-left')
+      goBackButton.classList.add('fa-angle-right')
+      goBackButton.style.opacity = '1'
+      goBackButton.title = l('goForward')
+    } else {
+      goBackButton.classList.remove('fa-angle-right')
+      goBackButton.classList.add('fa-angle-left')
+      goBackButton.style.opacity = '0.5'
+      goBackButton.title = l('goBack')
+    }
```

Comment:
Avoid repetition by moving all the `classList` modifications out of the if-clauses and use `toggle`:

```
goBackButton.classList.toggle('fa-angle-left')
goBackButton.classList.toggle('fa-angle-right')
```

Also, this code seems suspiciously copy-pasted from tabBar. Move it to a common function to keep common logic to avoid the need to support one code on two places.